cli/tests/unit/http_test.ts

```diff
@@ -1142,6 +1142,49 @@ Deno.test(
   },
 );
 
+// https://github.com/denoland/deno/pull/13628
+Deno.test(
+  {
+    ignore: Deno.build.os === "windows",
+    permissions: { read: true, write: true },
+  },
+  async function httpServerOnUnixSocket() {
+    const filePath = Deno.makeTempFileSync();
+
+    const promise = (async () => {
+      const listener = Deno.listen({ path: filePath, transport: "unix" });
+      for await (const conn of listener) {
+        const httpConn = Deno.serveHttp(conn);
+        for await (const { request, respondWith } of httpConn) {
+          assertEquals(
+            new URL(request.url).pathname,
+            "/path/name",
+          );
```

Comment:
Please check that the entire URL is what you expect (scheme + host too).

Reply:
Ok, I have added the test for other components.